pkg/kubelet/dockershim/convert.go

```diff
@@ -141,6 +144,9 @@ func toRuntimeAPISandboxState(state string) runtimeApi.PodSandBoxState {
 
 func toRuntimeAPISandbox(c *dockertypes.Container) (*runtimeApi.PodSandbox, error) {
 	state := toRuntimeAPISandboxState(c.Status)
+	if len(c.Names) == 0 {
+		return nil, fmt.Errorf("unexpected empty sandbox name: %+v", c)
```

Comment:
can this lead to inconsistencies like create 2 of the same container for a given pod? if a container doesn't show up in list, it's deemed non-existent right?